library/src/main/java/com/cube/storm/content/lib/helper/FileHelper.java

```diff
@@ -0,0 +1,102 @@
+package com.cube.storm.content.lib.helper;
+
+import android.text.TextUtils;
+import com.cube.storm.content.lib.Constants;
+import com.cube.storm.util.lib.debug.Debug;
+
+import java.io.BufferedInputStream;
+import java.io.BufferedOutputStream;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+
+/**
+ * Helper class for deleting files
+ *
+ * @author Callum Taylor
+ */
+public class FileHelper
```

Comment:
This file is copied from Storm developer library, and some file-related helper methods from content library have been moved into it. None of the code is "new"